docker-images/kafka/scripts/kafka_mirror_maker_2_run.sh

```diff
@@ -63,4 +63,4 @@ if [ -n "$STRIMZI_JAVA_SYSTEM_PROPERTIES" ]; then
     export KAFKA_OPTS="${KAFKA_OPTS} ${STRIMZI_JAVA_SYSTEM_PROPERTIES}"
 fi
 
-./kafka_connect_run.sh
+exec ./kafka_connect_run.sh
```

Comment:
No tini?

Reply:
`tini` is used inside the `kafka_connect_run.sh`, isn't it enough?

Reply:
Yep. The actual java process for Kafka connect is run in tini via the `kafka_connect_run.sh` script. When that script was being called directly (rather than being exec'd as in this fix) it was causing tini to be run as a child process, meaning tini would output the following warning in the logs:
```
[WARN  tini (41)] Tini is not running as PID 1 and isn't registered as
a child subreaper. Zombie processes will not be re-parented to Tini, so
zombie reaping won't work
```